msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -139,14 +139,17 @@ You're now ready to test your messaging extension running in Outlook on Windows
 
 As you test your messaging extension, you can identify the source (originating from Teams versus Outlook) of bot requests by the [channelId](https://github.com/Microsoft/botframework-sdk/blob/main/specs/botframework-activity/botframework-activity.md#channel-id) of the [Activity](https://github.com/Microsoft/botframework-sdk/blob/main/specs/botframework-activity/botframework-activity.md) object. When a user performs a query, your service receives a standard Bot Framework `Activity` object. One of the properties in the Activity object is `channelId`, which will have the value of `msteams` or `outlook`, depending from where the bot request originates. For more, see  [Search based messaging extensions SDK](/microsoftteams/platform/resources/messaging-extension-v3/search-extensions).
 
-### Outlook
-
-To preview your app running in Outlook on Windows desktop, open Outlook logged in with credentials for your test tenant. Click on **New Email**. Open the **More apps** flyout menu on the top ribbon. Your messaging extension will be listed. You can invoke it from there and use it just as you would while composing a message in Teams.
-
 ### Outlook on the web
 
 To preview your app running in Outlook on the web, log in to [outlook.com](https://www.outlook.com) using credentials for your test tenant. Click on **New message**. Open the **More apps** flyout menu on the bottom of the composition window. Your messaging extension will be listed. You can invoke it from there and use it just as you would while composing a message in Teams.
 
+### Outlook
+
+> [!IMPORTANT]
+> Refer to the latest updates on [Microsoft Teams - Microsoft 365 Developer Blog](https://devblogs.microsoft.com/microsoft365dev/) to check if Outlook on Windows desktop support for Teams personal apps is available to your test tenant.
+
+To preview your app running in Outlook on Windows desktop, open Outlook logged in with credentials for your test tenant. Click on **New Email**. Open the **More apps** flyout menu on the top ribbon. Your messaging extension will be listed. You can invoke it from there and use it just as you would while composing a message in Teams.
```

Comment:
Please convert this into a procedure.

**To preview your app running in Outlook on Windows desktop**

1. Access Outlook.
2. Log in with your test tenant credentials.
3. Click **New Email**.
.....

Reply:
Done